Docs/developer_guide/build_instructions/windows.md

```diff
@@ -8,7 +8,10 @@ Slicer relies on a number of large third-party libraries (such VTK, ITK, DCMTK),
 
 ## Install prerequisites
 
-- [CMake](https://www.cmake.org/cmake/resources/software.html) version that meets at least the minimum required CMake 3.21.1 <= version [< 3.25.0](https://gitlab.kitware.com/cmake/cmake/-/issues/24180) (or 3.16.3 <= version < 3.20.4)
+- [CMake](https://www.cmake.org/cmake/resources/software.html) version >= 3.16.3.
+  - Versions with known Slicer build issues:
+    - 3.20.4 to 3.21.0
+    - 3.25.0
```

Comment:
```suggestion
   - 3.25.0 and 3.25.1
```

https://github.com/Slicer/Slicer/blob/523cd95ff5cc614eed7722358628cf2e2a3e6eb4/CMakeLists.txt#L3-L5

Reply:
I guess since that is inclusive of 3.25.2 also having issues since the fix (https://github.com/Kitware/CMake/commit/771387523a02e0c1ef5f68d20e9efc210b19c868) was included in 3.25.3, then probably should be

```suggestion
    - 3.25.0 to 3.25.2
```